scss/_variables.scss

```diff
@@ -649,6 +649,14 @@ $box-shadow-inset:  null !default; // Boosted mod: instead of `inset 0 1px 2px r
 $component-active-color:      $black !default;
 $component-active-bg:         $primary !default;
 
+// scss-docs-start focus-ring-variables
+$focus-ring-width:      .25rem !default;
+$focus-ring-opacity:    .25 !default;
+$focus-ring-color:      rgba($primary, $focus-ring-opacity) !default;
+$focus-ring-blur:       0 !default;
+$focus-ring-box-shadow: null !default; // Boosted mod: instead of `0 0 $focus-ring-blur $focus-ring-width $focus-ring-color`
```

Comment:
Really hesitating whether to keep it or not 🤔

Reply:
I'd say let's keep it initialized to `null`
**[Bootstrap Side]** Wondering why no CSS var